internal/core/command/device.go

```diff
@@ -66,19 +77,33 @@ func executeCommandByDeviceID(
 		return "", errors.NewErrCommandNotAssociatedWithDevice(commandID, deviceID)
 	}
 
-	return executeCommandByDevice(ctx, d, c, body, queryParams, isPutCommand, lc)
+	return executeCommandByDevice(ctx, d, c, body, lc, r, httpCaller)
+}
+
+// Extract Device ID and Command ID from the HTTP request, returning an
+// error if either (Device ID or Command ID) is not successfully extracted.
```

Comment:
Since we have other changes that need to happen before we merge, can we update this to be a proper GoDoc comment?